### PR TITLE
fix(plugin-meetings): streamline rtpRecovered mqe

### PR DESCRIPTION
**Pull request by edvujic**
# COMPLETES [SPARK-537027](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-537027) #

## This pull request addresses

The `rtpRecovered` MQE incorrectly reports a default value of "0", failing to reflect the actual number of RTP packets recovered by FEC during an interval.

The metric should accurately count and report the number of lost RTP packets that FEC recovers.

https://confluence-eng-gpk2.cisco.com/conf/display/webexmedia/MQE+Metrics+Tables

### Change Type

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

- Verifies no RTP recovered packets are present initially.
- Confirms RTP recovered packets match FEC packets received.
- Ensures accurate RTP recovered packet count after discards.
- Checks RTP recovered packets reset to zero after intervals.

### I certified that

- [X] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [X] I discussed changes with code owners prior to submitting this pull request

- [X] I have not skipped any automated checks
- [X] All existing and new tests passed
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts [hunk 1]
```diff
@@ -69,6 +69,8 @@ export const getAudioReceiverMqa = ({
     (totalFecPacketsDiscarded - lastFecPacketsDiscarded);
   audioReceiver.common.fecPackets = fecRecovered;
 
+  audioReceiver.common.rtpRecovered = fecRecovered;
```
Comment (mkesavan13) [inline packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts:72]: This is just the same value we assign to a different variable. Is this correct? If so, what is the problem in using the existing one `fecPackets`?

Comment (edvujic) [reply packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts:72]: As it stands, FEC is the only method for audio packet recovery from the web client. This means that for audio receivers, the `fecPackets` count will directly reflect the `rtpRecovered` stats.

Comment (mkesavan13) [reply packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts:72]: I see. Got it.

**Review by mkesavan13 — APPROVED** [review]

File: packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts [hunk 2]
```diff
@@ -69,6 +69,8 @@ export const getAudioReceiverMqa = ({
     (totalFecPacketsDiscarded - lastFecPacketsDiscarded);
   audioReceiver.common.fecPackets = fecRecovered;
```
Comment (brycetham) [inline packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts:70]: Wasn't there a conversation in the "Ask MQA" space about `fecPackets` being incorrect here? From Nathan: "`fecPackets` in MQE is the number of FEC packets received, regardless of whether they were used in recovery or not. So this sounds like the WebRTC stats `fecPacketsReceived` field."

Should we fix that here?

Comment (edvujic) [reply packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts:70]: Ah yeah, good catch @brycetham

File: packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts [hunk 3]
```diff
@@ -67,7 +67,9 @@ export const getAudioReceiverMqa = ({
     totalFecPacketsReceived -
     lastFecPacketsReceived -
     (totalFecPacketsDiscarded - lastFecPacketsDiscarded);
-  audioReceiver.common.fecPackets = fecRecovered;
+  audioReceiver.common.fecPackets = totalFecPacketsReceived - lastFecPacketsReceived;
```
Comment (brycetham) [inline packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts:70]: Should we add unit tests for this?

**Review by brycetham — APPROVED** [review]

**Pull request merged**